package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "devDependencies": {
+    "strip-bom": "^3.0.0"
```

Comment:
wat

Reply:
I may have gotten a bit excited, sorry. The file content reading code I cut and pasted from StackOverflow was keen on UTF-8. I'll remove that.